### ISSUE TITLE
fatal error: 'Fabric.h' file not found

### ISSUE THREAD

**Issue by Sukendh** [kiwi-bop/flutter_crashlytics#57]
I'm not able to build in iOS after flutter upgrade. I'm not fabric, but I'm getting an error like this

`Running pod install...
Starting Xcode build...
Xcode build done.                                           22.9s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    === BUILD TARGET shared_preferences OF PROJECT Pods WITH CONFIGURATION Debug ===
    /Users/mobile-di/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_crashlytics-0.2.1/ios/Classes/FlutterCrashlyticsPlugin.m:2:9: fatal error: 'Fabric.h' file not found
    #import "Fabric.h"
            ^~~~~~~~~~
    1 error generated.`

here is my flutter doctor output

**Comment by Sukendh** [comment]
here is my flutter doctor output
`[✓] Flutter (Channel stable, v1.0.0, on Mac OS X 10.14.3 18D42, locale en-IN)
    • Flutter version 1.0.0 at /Users/mobile-di/flutter
    • Framework revision 5391447fae (3 months ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/mobile-di/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = /Users/mobile-di/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 1.9.4
    • CocoaPods version 1.5.2

[✓] Android Studio (version 3.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 33.0.1
    • Dart plugin version 182.5215
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[✓] VS Code (version 1.31.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.22.3

[✓] Connected device (1 available)`

**Comment by jaumard** [comment]
Strange, do you have the same error with the example app inside this module ? Can you try please ? maybe just a configuration problem...

**Comment by Sukendh** [comment]
Sure I will check in example app and will update soon.

**Comment by Sukendh** [comment]
when i added crashlytics in new sample app Im getting this.
`Xcode build done.                                           24.9s
Installing and launching...
2019-02-19 12:32:55.045 ios-deploy[5862:66021] [ !! ] Error 0xe80000be: This application's application-identifier entitlement does not match that of the installed application. These values must match for an upgrade to be allowed. AMDeviceSecureInstallApplication(0, device, url, options, install_callback, 0)
Could not install build/ios/iphoneos/Runner.app on d468c48cc31f1309cf13bc50d24011ccf6ffdde6.
Try launching Xcode and selecting "Product > Run" to fix the problem:
  open ios/Runner.xcworkspace
`

when i run in Xcode im getting this
` 'flutter_crashlytics/FlutterCrashlyticsPlugin.h' file not found`

**Comment by Sukendh** [comment]
Finally found this link [here which posted 2 years before](https://www.bountysource.com/issues/32678363-conflict-with-fabric). I replaces the

>  #import "Fabric.h"

 to  

> #import <Fabric/Fabric.h>

 . Its working fine now.

**Comment by jaumard** [comment]
Yeah your trouble come from the fact that you're adding crashlytics, but the plugin already does it, so you have 2 Fabric.h and it doesn't know which one to take.
